.github/workflows/build-graphscope-images-linux.yml

```diff
@@ -146,3 +155,355 @@ jobs:
         sudo docker push ${{ env.REGISTRY }}/graphscope/interactive-frontend:${tag}
         sudo docker push ${{ env.REGISTRY }}/graphscope/interactive-executor:${tag}
         sudo docker push ${{ env.REGISTRY }}/graphscope/learning:${tag}
+
+    - name: Clean Nightly Image
```

Comment:
we don't need to clean images during `x86` building process on `github runner` machine